lib/lotus/repository.rb

```diff
@@ -283,7 +283,8 @@ def persist(entity)
       #
       #   ArticleRepository.create(article) # no-op
       def create(entity)
```

Comment:
In this issue just update for `create` method so I haven't covered for `persist` yet. I'll do it when I rebase.